src/Raven.Server/Documents/Revisions/RevisionsStorage.cs

```diff
@@ -1589,9 +1595,15 @@ private static ByteStringContext.InternalScope GetLastKey(DocumentsOperationCont
         }
 
         [MethodImpl(MethodImplOptions.AggressiveInlining)]
-        private static unsafe ByteStringContext<ByteStringMemoryCache>.InternalScope GetKeyWithEtag(DocumentsOperationContext context, Slice lowerId, long etag, out Slice prefixSlice)
+        public static unsafe ByteStringContext<ByteStringMemoryCache>.InternalScope GetKeyWithEtag(DocumentsOperationContext context, Slice lowerId, long etag, out Slice prefixSlice)
```

Comment:
Why there are so many `private` to `public` changes here? It is _potentially_ dangerous because now the API encourages to use methods that were private previously. Private methods often have some assumptions about its usage.

Reply:
I had to do it cause I call them from the tests

Reply:
Then at least `internal` (it will work since we have `InternalsVisibleTo` in place)

Reply:
Done